src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core'
+import { IconButton, IconButtonSize } from '../../enums/icon-button'
+
+export interface MultiSelectOptions {
+  label: string
+  id: string
+  classesOnSelected?: string
+  classesOnList?: string
+}
+
+@Component({
+  selector: 'sofie-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MultiSelectComponent {
+  @Input() public options: MultiSelectOptions[]
+  @Input() public selectedOptionsIds: string[]
```

Comment:
This makes so it is not the MultiSelect itself that keeps tracks of what is selected. That seems to be a big responsibility of the MultiSelect that you have removed from it. 
Why is it not the MultiSelect that keeps track of what the MultiSelect has selected?